pydataapi/pydataapi.py

```diff
@@ -14,6 +14,7 @@
 )
 
 import boto3
+from more_itertools import chunked, flatten
```

Comment:
Adding this package has made my chalice lambda go above 50MB 😆 I thought it would be funny to share 🤣

Reply:
Really?
The package looks small which is 57.3 kB 🤔 
https://pypi.org/project/more-itertools/#files

Reply:
> Really?
> The package looks small which is 57.3 kB 🤔
> https://pypi.org/project/more-itertools/#files

Yeah. I think my lambda was very very very close to 50MB. I will have to clean some things out if I can.

Reply:
OK, I will create a new issue to reduce the size of dependency packages.